hudi-client/hudi-flink-client/src/main/java/org/apache/hudi/client/HoodieFlinkWriteClient.java

```diff
@@ -372,6 +372,8 @@ public void completeCompaction(
     } finally {
       this.txnManager.endTransaction(Option.of(compactionInstant));
     }
+    WriteMarkersFactory.get(config.getMarkersType(), table, compactionCommitTime)
+      .quietDeleteMarkerDir(context, config.getMarkersDeleteParallelism());
```

Comment:
Should this happen just before completing inflight compaction (`CompactHelpers.getInstance().completeInflightCompaction(table, compactionCommitTime, metadata);`) for readability?

Reply:
Thank you for your review. Generally, Delete the remaining temporary marker files in postCommit stage. So, I think compaction progress also need to delete marker files after completing inflight compaction.

Reply:
I answered your questions in the comments and hope you can review my code again.

Reply:
The modification is the same as that of SparkRDDWriteClient.

Reply:
Sg.